packages/jest/src/resultsProcessor.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { addResult, createEmptyTestResult } from '@jest/test-result';
+import { AggregatedResult, AssertionResult, TestResult } from '@jest/test-result/build/types';
```

Comment:
Can we not import these types from the `@jest/test-result` entry?

Reply:
Yes we can :) Thanks for catching that